app/views/admin/index.scala.html

```diff
@@ -263,82 +259,82 @@ <h1>Coverage</h1>
                                 </tr>
                                 <tr>
                                     <th>Audited Streets</th>
-                                    <td>@StreetEdgeTable.countAuditedStreets()
-                                        (@("%.0f".format(StreetEdgeTable.auditCompletionRate(1) * 100))%)</td>
-                                    <td>@StreetEdgeTable.countAuditedStreets(1, "All", true)
-                                        (@("%.0f".format(StreetEdgeTable.auditCompletionRate(1, "All", true) * 100))%)</td>
-                                    <td>@StreetEdgeTable.countTotalStreets()</td>
+                                    <td><span class="Coverage-audited-streets-totals" id="all-users-value"></span>
```

Comment:
You'll want your HTML ids to be unique, instead of needing to use a CSS class to fully identify an element. CSS should generally be reserved for styling, and I don't think any of these classes do that. We should just use more descriptive IDs, like "audited-streets-all-users" / "audited-streets-registered" / "distance-all-users".